### PR TITLE
COCICCS-6024:Edit safe:Application name selected is not getting displayed

### PR DESCRIPTION
**Pull request by msivatmob**

### CODE REVIEW

**Review by sarathmohan4 — DISMISSED** [review]

File: tvaultapi/src/main/java/com/tmobile/cso/vault/api/utils/SafeUtils.java [hunk 1]
```diff
@@ -39,6 +39,7 @@
 import com.tmobile.cso.vault.api.exception.LogMessage;
 import com.tmobile.cso.vault.api.model.Safe;
 import com.tmobile.cso.vault.api.model.SafeBasicDetails;
+import com.tmobile.cso.vault.api.model.SafeSecretCount;
```
Comment (alexrajan3) [inline tvaultapi/src/main/java/com/tmobile/cso/vault/api/utils/SafeUtils.java:42]: Not used anywhere  - Remove this unwanted import

File: tvaultapi/src/main/java/com/tmobile/cso/vault/api/service/SafesService.java [hunk 2]
```diff
@@ -26,6 +26,7 @@
 import com.tmobile.cso.vault.api.validator.TokenValidator;
 import org.apache.commons.lang3.ArrayUtils;
 import org.apache.commons.lang3.StringUtils;
+//import org.springframework.util.StringUtils;
```
Comment (alexrajan3) [inline tvaultapi/src/main/java/com/tmobile/cso/vault/api/service/SafesService.java:29]: Remove unwanted commented code

**Review by alexrajan3 — CHANGES_REQUESTED** [review]
Please check the review comments

File: tvaultapi/src/main/java/com/tmobile/cso/vault/api/utils/SafeUtils.java (hunk 1 above)
Comment (msivatmob) [reply tvaultapi/src/main/java/com/tmobile/cso/vault/api/utils/SafeUtils.java]: Removed the import.

File: tvaultapi/src/main/java/com/tmobile/cso/vault/api/service/SafesService.java (hunk 2 above)
Comment (msivatmob) [reply tvaultapi/src/main/java/com/tmobile/cso/vault/api/service/SafesService.java]: Removed the commented line.

**Pull request merged**